examples/mspaint.py

```diff
@@ -9,14 +9,30 @@
 JPEG image and resizes it using "Resize and Skew" dialog.
 """
 
+from __future__ import print_function
+import logging
+import sys
```

Comment:
You have static analyzer warning here: sys in unused. Please remove this import.